platform-sdk/swirlds-common/src/main/java/com/swirlds/common/startup/CommandLineArgs.java

```diff
@@ -26,11 +26,11 @@
  *
  * @param localNodesToStart the set of nodes to start on this machine
  */
-public record CommandLineArgs(@NonNull Set<NodeId> localNodesToStart) {
-    /**
-     * The command line option to start a set of nodes on this machine
-     */
+public record CommandLineArgs(@NonNull Set<NodeId> localNodesToStart, boolean pcesRecovery) {
+    /** The command line option to start a set of nodes on this machine */
     public static final String OPTION_LOCAL = "-local";
+    /** When set, the platform will perform a PCES recovery and shut down */
+    public static final String OPTION_PCES_RECOVERY = "-pces-recovery";
```

Comment:
I'm not sure adding new command line arguments parsed by the browser is the right approach. We have been attempting to move away from the browser being responsible for parsing command line stuff.

Could we make this a configuration option instead? Or perhaps just a constructor argument to the browser?

Reply:
I don't like the configuration option, its really not a config, its a special mode of operation. Maybe we can discuss the best option today.

Reply:
based on the discusstion, it will be an agrument in the method only

Reply:
done